app/services/theme.ts

```diff
@@ -91,11 +91,7 @@ export default class Theme extends Service {
         let pathPrefix = '/';
 
         if (!this.isDomain) {
-            pathPrefix += `${this.settings.routePath}/`;
-
-            if (this.isProvider) {
```

Comment:
Super minor,  on line 89 `@computed('id', 'isDomain', 'isProvider', 'settings.routePath')`, you can remove `isProvider` as that property no longer impacts this getter